dubbo-serialization/dubbo-serialization-kryo/src/main/java/org/apache/dubbo/common/serialize/kryo/CompatibleKryo.java

```diff
@@ -34,7 +38,7 @@ public Serializer getDefaultSerializer(Class type) {
             throw new IllegalArgumentException("type cannot be null.");
         }
 
-        if (!type.isArray() && !type.isEnum() && !ReflectionUtils.checkZeroArgConstructor(type)) {
+        if (!type.isArray() && !type.isEnum() && !type.equals(LocalDate.class) && !type.equals(LocalDateTime.class) && !type.equals(LocalTime.class) && !ReflectionUtils.checkZeroArgConstructor(type)) {
```

Comment:
Simply check types will not solve the problem.

Reply:
Sorry, I've checked that this works, and kryo has internally registered some Serializers for Java 8 Time types. 

But we still have a problem, we cannot list all possible classes here.